theme/header.html

```diff
@@ -0,0 +1,28 @@
+<header class="header">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-9">
+                <h1 class="header__title">
+                    Documentation
+
+                    {% if config.site_name != 'Documentation' %}
+                        <small class="header__subtitle">{{ config.site_name }}</small>
+                    {% endif %}
+                </h1>
+            </div>
+            <div class="col-md-3">
+                {% if config.site_name != 'Documentation' %}
+                    <a href="#" class="header__link header__link--search" data-toggle="modal" data-tooltip="" data-placement="bottom" data-target="#mkdocs_search_modal" title="Open search">
+                        <i class="fa fa-search"></i> Search
+                    </a>
+                    <a href="https://docs.zendframework.com" class="header__link" data-tooltip data-placement="bottom" title="Show component list">
+                        <i class="fa fa-book"></i> Components
+                    </a>
+                    <a href="{{ config.repo_url }}" class="header__link header__link--github" data-tooltip data-placement="bottom" title="Go to repository of {{ config.site_name }}">
+                        <i class="fa fa-github"></i> GitHub
+                    </a>
```

Comment:
For some reason this is under search and components. I'm guessing it should be on the same line behind?

Maybe you can fix it by making the header__title `col-md-6` and the headerlinks parent also `col-md-6` and right align these (`text-right` or `pull-right` ???).